src/backend/gl/src/lib.rs

```diff
@@ -553,6 +553,7 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
             .0
             .legacy_features
             .contains(info::LegacyFeatures::SRGB_COLOR)
```

Comment:
can the `LevacyFeatures::SRGB_COLOR` bit already take into account that it's not available on ES? seems strange that we have this `is_embedded` check in addition to it.
Also, something tells me that ES really does support SRGB. Maybe not all ES?

Reply:
Based on info in [this extension](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_sRGB_write_control.txt), `FRAMEBUFFER_SRGB` is always enabled with `OpenGL ES 3.0`. Added extra comment to code.